### ISSUE TITLE
Integrate Dart analysis server

### ISSUE THREAD

**Issue by austincummings** [austincummings/atom-dart#3]

**Comment by sethladd** [comment]
Is this still planned? If so, reach out to analyzer-discuss@dartlang.org list if you have questions.

**Comment by kaendfinger** [comment]
Yes. Austin has been working on a separate branch to port this to my Atom.dart library. https://github.com/austincummings/atom-dart/tree/dart

I wrote an analysis client as well: https://github.com/DirectMyFile/scraps/blob/master/io/analysis.dart

It's pluggable enough to use the process API in Atom.dart to provide the Analysis Server stuff :)

Thanks for being concerned. I have not been able to be involved in much code writing for this yet (as far as the actual plugin goes), but once I'm free, I'll make sure to help @austincummings make an amazing Dart plugin.

**Comment by austincummings** [comment]
@sethladd Yep, still planned. Haven't had much time at all lately to work on it. That should be changing soon though.

@kaendfinger I'll add you as a contributor so we can start development.
